src/todo/UI.lua

```diff
@@ -132,6 +150,127 @@ function todo.create_task_table(frame, player)
     return table
 end
 
+function todo.create_export_dialog(player)
```

Comment:
```suggestion
function todo.create_export_frame(player)
```

Here you use dialog, the rest of the code use frame for the same thing. Maybe we should keep to frame or change everthing else to dialog. IMO both are good description for the pop-up window.

Reply:
Yeah I also realised this is inconsistent..
My thinking was "Frame" is something like the main frame and other static frames.
Dialogs are popups that are only used for a short amount of time and block gui.center.

I thought about doing another major rework of everything including abstracting all this manual "lets create the same dialog stub over and over again" away. Then we would also have a more consistent naming.